packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -198,6 +200,32 @@ const genFullBackCompatConfig = (): CompatConfig[] => {
 	return _configList;
 };
 
+export function isCompatVersionBelowMinVersion(
```

Comment:
Can you add a description to this function and comments inside the function. It's very hard to follow what is happening.